### DISCUSSION TITLE
MIMIC-iv 2.2: Some question about sofa_time in 'sepsis' table

### DISCUSSION THREAD

**Discussion by Alanthu** [MIT-LCP/mimic-code#1525] [MIMIC-IV]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Hello my questions is based on MIMIC-IV, version 2.2:

Currently I'm trying on early detection of sepsis using data from MIMIC-IV. However I met some problem when calculating the time interval between admit_time and sofa_time(which is considered to be the onset time of sepsis). I found that sofa_time for some patient ( about 28 in total) are earlier than admit_time and I'm wondering why would this happend.

Thank you for your time and help.

**Comment by alistairewj** [comment]
There's a few reasons this could happen.

- admittime could be recorded late; i.e. the patient was already in the hospital and the administratively recorded "admission" time is later than when they actually physically were in the hospital
- sofa time could be early; since SOFA is dependent on lab values, it could be they had an outpatient lab that was abnormal, or they had a lab measurement in the emergency department that was abnormal (the emergency department is considered pre-admission)
